lib/python/test/testsuite/Math/deviation.py

```diff
@@ -0,0 +1,208 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2022 PIConGPU contributors
+Authors: Mika Soren Voss
+License: GPLv3+
+
+This module contains some auxiliary functions to determine the
+deviation between theory and simulation. It is not yet complete
+in the present version and will be expanded over time.
+
+Routines in this module:
+
+getDifference(theory, simulation)->float or array:
+    calculates the difference between the theory value and
+    the simulation value
+
+getMaxDifference(theory, simulation):
+    calculates the maximum deviation between theory and simulation
+
+getDifferenceInPercentage(theory, simulation):
+    calculates the difference between the theory value and
+    the simulation value in percentage
+
+getAcceptanceRange(theory, acceptance:float = None):
+    represents the range around the theoretical value,
+    which as a limit leads to the acceptance of the test
+
+getTestResult(theory, simulation, acceptance:float = None) -> bool:
+    determines whether a test passes or fails based on the acceptance range
+"""
+
+__all__ = ["getDifference",
+           "getMaxDifference",
+           "getDifferenceInPercentage",
+           "getAcceptanceRange",
+           "getTestResult"]
+
+import testsuite._checkData as cD
+import numpy as np
+
+
+def getMaxDifference(theory, simulation):
+    """
+    calculates the maximum deviation between theory and simulation
+
+    Calculates:
+    --------
+    theory -simulation
+
+    Input:
+    -------
+    theory :     float or array
+                 theoretical value
+                 Note: if list and the values
+                 of the simulations are also a list, both must have
+                 the same length
+
+    simulation : float or array
+                 Values from the simulation
+                 Note: if the list and the values
+                 of the theory are also a list,
+                 both must have the same length
+
+    Return:
+    -------
+    out : The maximum in the deviation
+    """
+
+    difference = np.abs(theory - simulation)
+
+    return min(difference)
+
+
+def getDifference(theory, simulation):
+    """
+    calculates the difference between the theory value and
```

Comment:
```suggestion
    calculates the absolute difference between the theory value and
```

perhaps also indicated this is the method name